Automattic/newspack-theme
# fix: remove translation workaround; the issue is fixed in WP

### All Submissions:


### Changes proposed in this Pull Request:

In https://github.com/Automattic/newspack-theme/pull/2411, we tweaked how translations are loaded to work around a bug in WordPress 6.7. Now that that bug's been fixed in 6.7.1, we can rollback this change.

It looks like translations are still loaded in 6.7.1 with the fix in place, so I'm merging this into trunk instead of handling as a hotfix again.

### How to test the changes in this Pull Request:

1. Make sure your test site is running 6.7.1 (released Nov 21).
2. Apply this PR.
3. Switch your site to one of the languages bundled in the theme (like Spanish or German).
4. Navigate to a single post or an archive page and confirm that the byline prefix ('by') is being translated.

### Other information:
